clients/unshaded/src/main/java/tachyon/client/file/TachyonFileSystemFactory.java

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
I think the convention in our code base is to define the Factory as a nested class of the class it generates. So this code should go into TachyonFileSystem.